### PR TITLE
Draws RNA sequences via DrawRNA-js

### PR DESCRIPTION
**Pull request by yhoogstrate**
Visualization for RNA 2D structrures from "rna_eps" and "fasta" format.

### CODE REVIEW

File: visualisations/drawrnajs/templates/drawrnajs.mako [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<script type="text/javascript" src="../../../../../static/scripts/libs/d3.js"></script>
+<script type="text/javascript" src="../../../../../static/scripts/libs/jquery/jquery.js"></script>
+<script type="text/javascript" src="../../../../../static/scripts/libs/jquery/jquery.migrate.js"></script>
+
+
+<meta charset="UTF-8">
+<link type="text/css" rel="stylesheet" href="http://workmen.biojs.net/github/bene200/drawrnajs/master/style/style.css">
+<link type="text/css" rel="stylesheet" href="http://workmen.biojs.net/github/bene200/drawrnajs/master/style/spectrum.css">
```
Comment (bgruening) [inline visualisations/drawrnajs/templates/drawrnajs.mako]: Can we add these files locally?

```
<%
    root = h.url_for( '/' )
%>

${h.stylesheet_link( root + 'plugins/visualizations/drawrnajs/static/css/style.css' )}
...
```

File: visualisations/drawrnajs/templates/drawrnajs.mako [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+<script src="https://wzrd.in/bundle/drawrnajs@0.3.4"></script>
+
```
Comment (bgruening) [inline visualisations/drawrnajs/templates/drawrnajs.mako]: Would be awesome if we could store this locally as well.

**Comment by bgruening** [comment]
@yhoogstrate amazing! Thanks! 
If you can please try to store all dependencies locally. Otherwise, we can merge it and track this as issue.

**Comment by yhoogstrate** [comment]
Everything has been moved to local directories including the scripts that were in the template (+ grunt uglify to make the static files smaller).

**Comment by bgruening** [comment]
Awesome! I promised @bene200 to get this in and now you did it! Absolutely awesome. I will get this running on our Freiburger Galaxy Server as soon as possible. 
@bene200 expect some screenshots soon.

**Comment by bene200** [comment]
This is extremely cool! Really looking forward to seeing the results!!

**Comment by yhoogstrate** [comment]
@bgruening @bene200 Thanks for the help :+1: . Let's see if there are other file formats applicable for this, like CT and DotBracket Fasta:

*>  A stem-loop structure
GGGCGAAUUGGGUACCGGGCCC
((((...((((...))))))))*

**Comment by bgruening** [comment]
rna_vis

Awesome! We have a few issues with the file formats but it's mostly working.

**Pull request merged**